docs/source/guide/helm_values.md

```diff
@@ -34,166 +34,173 @@ meta_description: For cases when you want to customize your Label Studio Enterpr
-| `global.redisConfig.password.secretKey`                                     | string | ""                                  | Key of an existing secret holding the password of Redis database                                    |
+| `global.imagePullSecrets`                                                   | string | []                                  | Image pull secret to use for registry authentication.                                                |
+| `global.image.repository`                                                   | string | heartexlabs/label-studio-enterprise | Image repository.                                                                                    |
+| `global.image.pullPolicy`                                                   | string | IfNotPresent                        | Image pull policy.                                                                                   |
+| `global.image.tag`                                                          | string | IfNotPresent                                  | Image tag (immutable tags are recommended).                                                          |
+| `global.djangoConfig.db`                                                    | string | default                             | Django default config name.                                                                          |
+| `global.djangoConfig.settings_module`                                       | string | htx.settings.label_studio           | Django default settings module name.                                                                |
+| `global.enterpriseLicense.secretName`                                       | string | htx.settings.label_studio                                 | Name of an existing secret holding the Label Studio Enterprise license information.                  |
+| `global.enterpriseLicense.secretKey`                                        | string | license                             | Key of an existing secret holding the enterprise license information.                               |
+| `global.pgConfig.host`                                                      | string | license                                  | PostgreSQL hostname.                                                                                 |
+| `global.pgConfig.port`                                                      | string | 5432                                | PostgreSQL port.                                                                                    |
+| `global.pgConfig.dbName`                                                    | string | 5432                                  | PostgreSQL database name.                                                                            |
```

Comment:
5443 everywhere

Reply:
This is addressed in our Slack convo
https://heartex.slack.com/archives/D03JUMAH9ED/p1662494050183749